core/src/banking_stage/vote_worker.rs

```diff
@@ -377,100 +374,58 @@ impl VoteWorker {
         bank_creation_time: &Instant,
         transactions: &[impl TransactionWithMeta],
     ) -> ProcessTransactionsSummary {
-        let mut chunk_start = 0;
```

Comment:
Getting ride of chunking greatly simplifies this function 👍🏼 Just wondering if there are cases that sending a large batch of votes could have negative impact on network? Actually not even sure if batch will be large enough, do you have test data to indicate how batch size changes with this pr?

Reply:
There's not a functional difference w/ the change in terms of batch size, since we still split into chunks [here](https://github.com/anza-xyz/agave/blob/master/core/src/banking_stage/vote_worker.rs#L219).

`UNPROCESSED_BUFFER_STEP_SIZE` is equal to `TARGET_NUM_TRANSACTIONS_PER_BATCH`.
So we already had chunks that were the size of the inner loop.

Reply:
arrh! thanks for pointing this out.